ArchiSteamFarm/WebBrowser.cs

```diff
@@ -267,6 +267,26 @@ internal sealed class WebBrowser : IDisposable {
 			return null;
 		}
 
+		internal async Task<HtmlDocumentResponse> UrlPostMultipartFormToHtmlDocument(string request, IEnumerable<KeyValuePair<string, string>> data, byte maxTries = MaxTries) {
+			if (string.IsNullOrEmpty(request) || (maxTries == 0)) {
+				ArchiLogger.LogNullError(nameof(request) + " || " + nameof(maxTries));
+				return null;
+			}
+
+			MultipartFormDataContent content = new MultipartFormDataContent();
+			foreach (KeyValuePair<string, string> kvp in data) {
+				content.Add(new StringContent(kvp.Value), kvp.Key);
+			}
+
+			HttpResponseMessage responseHttp = await HttpClient.PostAsync(request, content).ConfigureAwait(false);
```

Comment:
This is entirely unacceptable, you should use `InternalPost`, modifying it to match your requirement if needed. Only `InternalRequest` should ever access `HttpClient` object. Your request will fail entirely for redirections, lost sessions and everything else that you're not handling.